app/views/devise/sessions/_return_to_service_provider.html.slim

```diff
@@ -1,4 +1,5 @@
-.sm-col.py1.sm-p0
-  = link_to "‹ #{t('links.back_to_sp', sp: decorated_session.sp_name)}",
-    decorated_session.sp_return_url,
-    class: 'inline-block truncate align-bottom sm-maxw-190p'
+- if decorated_session.sp_return_url.present?
```

Comment:
If Jessie were watching our PRs, she'd tell us to avoid logic/branching in partials. Since the entire partial is being wrapped in an if-statement, should we just move the logic that renders this partial into a decorator?

Is this separate than the `return_to_sp_from_start_page_partial` method?